kbase-extension/static/kbase/js/widgets/appWidgets2/input/floatInput.js

```diff
@@ -142,7 +130,10 @@ define([
                                         message: result.errorMessage,
                                     });
                                     ui.setContent('input-container.message', message.content);
-                                    message.events.attachEvents();
+                                    // FIXME: when enabled, this (silently) throws error
+                                    // "Error: could not find node for #kb_html_.+"
+                                    //
+                                    // message.events.attachEvents();
```

Comment:
commenting this out with a message as I didn't want to delve too far into implementation bugs